src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -33,11 +32,13 @@
 
 int RemoteTaskCheckInterval = 100; /* per cycle sleep interval in millisecs */
 int TaskExecutorType = MULTI_EXECUTOR_ADAPTIVE; /* distributed executor type */
-bool BinaryMasterCopyFormat = false; /* copy data from workers in binary format */
 bool EnableRepartitionJoins = false;
 
+/* deprecated GUC*/
+bool BinaryMasterCopyFormat = false;
 
-static bool HasReplicatedDistributedTable(List *relationOids);
+/* deprecated GUC*/
+int MaxAssignTaskBatchSize = 0;
```

Comment:
can move these into shared_libary_init.c as static variables.

(we could maybe just have one integer that we use for all such GUCs)